server/libs/userlibs.js

```diff
@@ -1,11 +1,11 @@
 const User = require ('../DataBase/models/user');
-const Token = require('../DataBase/models/tokens');
 const bcrypt = require ('bcryptjs');
 const jwt = require('jsonwebtoken');
-const {fieldsValidate} = require('./../utilities/utilityFunctions')
-const {statusCodes, messages, secretKeys} = require ('../utilities/constants');
+const {fieldsValidate, decodeToken} = require('./../utilities/utilityFunctions');
+const {createToken} = require('./tokenlibs');
+const {statusCodes, messages, secretKeys, timeScale} = require ('../utilities/constants');
 
-const register = function(userData) {
+const register = function (userData) {
     return new Promise((resolve,reject) => {
         let result = fieldsValidate(userData);
         if (result) {
```

Comment:
if you are not using result any where else to if check you can do if (fieldsValidate(userData))

Reply:
I am using result variable after if to send as message

Reply:
then change the name of that it should not be result